lib/collection/src/tests/sparse_vectors_validation_tests.rs

```diff
@@ -0,0 +1,197 @@
+use std::collections::HashMap;
+
+use segment::data_types::vectors::{NamedSparseVector, NamedVectorStruct, Vector, VectorStruct};
+use sparse::common::sparse_vector::SparseVector;
+use validator::Validate;
+
+use crate::operations::point_ops::{Batch, PointStruct, PointsBatch, PointsList};
+use crate::operations::types::{
+    BaseGroupRequest, ContextExamplePair, DiscoverRequestInternal, QueryEnum, RecommendExample,
+    RecommendRequestInternal, SearchGroupsRequestInternal, SearchRequestInternal,
+};
+use crate::operations::vector_ops::PointVectors;
+
+fn wrong_sparse_vector() -> SparseVector {
+    SparseVector {
+        indices: vec![1, 2],
+        values: vec![0.0, 1.0, 2.0],
+    }
+}
+
+fn wrong_named_vector_struct() -> NamedVectorStruct {
+    NamedVectorStruct::Sparse(NamedSparseVector {
+        name: "sparse".to_owned(),
+        vector: wrong_sparse_vector(),
+    })
+}
+
+fn wrong_point_struct() -> PointStruct {
+    let vector_data: HashMap<String, Vector> =
+        HashMap::from([("sparse".to_owned(), wrong_sparse_vector().into())]);
+    PointStruct {
+        id: 0.into(),
+        vector: VectorStruct::Multi(vector_data),
+        payload: None,
+    }
+}
+
+fn wrong_recommend_example() -> RecommendExample {
+    RecommendExample::Sparse(wrong_sparse_vector())
+}
+
+fn check_validation_error(result: Result<(), ::validator::ValidationErrors>) {
+    println!("{:?}", result);
```

Comment:
can be remove now :)

Reply:
Thanks for this comment. I had an annoying integration test failure with a non-matched string of the error message. I spend one hour of fixing and forgot about removing debug stuff. Thanks for help!